client/scss/components/forms/_form-width.scss

```diff
@@ -0,0 +1,48 @@
+@use 'sass:map';
+@use 'sass:math';
+
+/**
+ * Calculate the ideal form width to avoid any overlap with the side panels.
+ */
+
+@mixin form-width-vw($breakpoint) {
+  .side-panel-open & {
```

Comment:
probably not a major issue for this PR but I think we might want to revisit this - might be nice to see if we can find a better way to have these two component's styles work in isolation.

Reply:
Yes, we’re also going to change the panels’ widths and breakpoints so the magic in here isn’t as necessary.